parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -1218,6 +1219,13 @@ prettyParseError s = \case
     , "but there wasn't one.  Maybe check your indentation:\n"
     , tokenAsErrorSite s tok
     ]
+  go Parser.EmptyMatch = mconcat
```

Comment:
I decided to add this corner case handling to the parser since I ran up against it in my testing.